homeassistant/components/ai_speaker/config_flow.py

```diff
@@ -0,0 +1,65 @@
+"""Config flow to configure AI-Speaker."""
+import logging
+
+from aisapi.ws import AisWebService
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers import aiohttp_client
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+AIS_CONFIG = {
+    vol.Required(CONF_HOST): str,
+}
+
+
+class AisConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a AI-Speaker config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_PUSH
+    DOMAIN = DOMAIN
+
+    async def async_step_user(self, user_input=None):
+        """Commissioning the configuration by the user."""
+        return self.async_show_form(step_id="confirm")
+
+    async def async_step_confirm(self, user_input=None):
+        """User confirmation step."""
+        if user_input is not None:
+            return self.async_show_form(
+                step_id="settings", data_schema=vol.Schema(AIS_CONFIG)
+            )
+
+        return self.async_show_form(step_id="confirm")
+
+    async def async_step_settings(self, user_input=None):
```

Comment:
If we just need to ask for the host address we can do that in a single step, the user step. Please simplify the config flow. Please use the template that the scaffold script creates as a base.